### ISSUE TITLE
Pangeo 2019 Community Meeting Announcement

### ISSUE THREAD

**Issue by jhamman** [pangeo-data/pangeo#604]
Details for the 2019 Pangeo Community Meeting are now available.

***tldr; please register ASAP so we can get a sense for the number of attendees.***

The meeting is open to anyone involved or interested in Pangeo. We encourage active Pangeo developers and new comers alike to register. A final agenda for the meeting will be available later this summer.

#### Meeting Details
- **Meeting webpage**: http://pangeo.io/meetings/2019_summer-meeting.html
- **Dates**: 21-23 August 2019
- **Location**: [Alder Hall](https://www.washington.edu/maps/#!/ald), Room 107, University of Washington campus, Seattle, Washington
- **Registration**: [Register Here](https://docs.google.com/forms/d/e/1FAIpQLSemu3o7Noo-B620tvIhIVrDCJHiBjYaTRin3RlBd-PN_CnVFw/viewform?usp=sf_link)

xref: https://github.com/pangeo-data/pangeo/issues/573
cc @pangeo-data/pangeo-members

**Comment by rabernat** [comment]
I think if we are serious about inviting newcomers, we need to advertise more widely. 

However, maybe there are logistical limits that require us to limit the number of attendees?

**Comment by nbren12** [comment]
This is great! So happy you are doing it in Seattle. If you want I could advertise this around UW. There are lots of xarray (and some dask) aficionados here.

**Comment by martindurant** [comment]
The 12-people limit seems to be getting prohibitive (I was late and couldn't get in), where could we discuss alternatives?

**Comment by rabernat** [comment]
@martindurant - here we are discussing a F2F meeting in August.

However, I hear your comment about the weekly checkin meetings.

The alternative is to use Zoom, which is not as nice for participants as appear.in, but, thanks to Columbia's institutional subscription, can accommodate hundreds of attendees.

**Comment by mmccarty** [comment]
Oh!  I didn't realize space was that limited.  I would love to attend to see how my org could learn from what this group has done and hopefully contribute back in some way.  Although, I'm totally fine giving up my spot to someone else more aligned with academia.

**Comment by rabernat** [comment]
Folks we are talking about **two separate things here**:

- The pangeo weekly video checkins on appear.in, limited to 12 people
- The pangeo annual IRL meeting in Seattle. Not sure the limitations here. Maybe @scottyhq can give us some details.

**Comment by martindurant** [comment]
(sorry, I think I'm the one that crossed the streams here)

**Comment by scottyhq** [comment]
Thanks @rabernat - I agree we should get newcomers to participate so please point people to the meeting webpage. We have a room reserved for up to 50 people.

**Comment by jhamman** [comment]
Sorry to be late the party here. My thought is that we have advertised here on Github and in the Pangeo google group. We already have nearly 30 ppl registered. I certainly want new comers but I'm nervous about over advertising at this point. If it were between sending out a bunch of tweets and emphasizing personal invitations, I think I'd prefer the latter.
